mixer/pkg/runtime/testing/data/data.go

```diff
@@ -467,6 +487,30 @@ spec:
     - ihalt1.thalt.istio-system
 `
 
+// RuleCheckOutput1 is a standard testing instance config with name rcheckoutput1. It references I1 and H1.
+var RuleCheckOutput1 = `
+apiVersion: "config.istio.io/v1alpha2"
+kind: rule
+metadata:
+  name: rcheckoutput1
+  namespace: istio-system
+spec:
+  actions:
+  - handler: hcheckoutput1.acheckoutput
+    instances:
+    - icheckoutput1.tcheckoutput.istio-system
+    name: sample
```

Comment:
A rule has several handler application. You might have two handlers in a single rule that would share the name for the result of the application and result in an ambiguity.

I should add that pieces of this PR are falling from the desire to add more expressive logical combinators in rule. Currently, rules are implicitly always ANDed, preventing blacklisting, for example. At one of the policy WG meetings, we have converged on expanding the existing rule to support ORs and other expressions, instead of adding another rule-like document to the config. That would also require having a name for each `handler(instances...)` in the rule.